raiden-ts/src/transport/epics/webrtc.ts

```diff
@@ -80,38 +107,39 @@ async function getMatrixIceServers(matrix: MatrixClient): Promise<RTCIceServer[]
   return servers;
 }
 
-// creates a filter function which filters valid MatrixEvents
-function filterMatrixVoipEvents<
-  T extends 'm.call.invite' | 'm.call.answer' | 'm.call.candidates' | 'm.call.hangup'
->(type: T, sender: string, callId: string, httpTimeout?: number) {
-  type ContentKey = T extends 'm.call.invite'
-    ? 'offer'
-    : T extends 'm.call.answer'
-    ? 'answer'
-    : T extends 'm.call.candidates'
-    ? 'candidates'
-    : never;
-  const contentKey = (type === 'm.call.invite'
-    ? 'offer'
-    : type === 'm.call.answer'
-    ? 'answer'
-    : type === 'm.call.candidates'
-    ? 'candidates'
-    : undefined) as ContentKey | undefined;
-  return (
-    // FIXME: remove any when MatrixEvent type exposes getAge & getContent methods
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    event: any,
-  ): event is MatrixEvent & {
-    getType: () => T;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    getContent: () => { call_id: string } & { [K in ContentKey]: any };
-  } =>
-    event.getType() === type &&
-    event.getSender() === sender &&
-    event.getContent()?.call_id === callId &&
-    (!httpTimeout || event.getAge() <= (event.getContent()?.lifetime ?? httpTimeout)) &&
-    (!contentKey || !!event.getContent()?.[contentKey]);
+type ExtMatrixEvent = MatrixEvent & {
+  getType: () => string;
+  getContent: () => { msgtype: string; body: string };
+};
+
+// returns a stream of filtered, valid MatrixEvents
+function matrixVoipEvents$<T extends 'offer' | 'answer' | 'candidates' | 'hangup'>(
```

Comment:
Should this be renamed now that we don't use the matrix VOIP functionality any more?

Reply:
Yes, maybe `matrixWebrtcEvents$` would be a better fit. I'll adapt. Thanks for the suggestion.